Orion.Net.Client/Configuration/Connector.cs

```diff
@@ -93,25 +93,27 @@ private BaseClientScript GetCommand(string commandTitle)
         {
             // Send result object to the correct uri :
             var dataUri = string.Empty;
-            HttpContent content = null;
+            HttpContent content = new StringContent(JsonConvert.SerializeObject(result), Encoding.UTF8, "application/json");
```

Comment:
Please keep the new StringContent in the switch case as some case could generate Something else than a StringContent object.